<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мироненко | ИП-811</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="photo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>Фото</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUploadPhoto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Загрузить фото</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReady">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 162);
color: brack;
</string>
    </property>
    <property name="text">
     <string>Готово</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>301</width>
      <height>172</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Дата рождения</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="dateOfBirth">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>E-mail</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButtonMan">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>М</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonWoman">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ж</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Должность</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Пол</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="linePost">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEmail">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineName">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ФИО</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="photoPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь до файла</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>btnUploadPhoto</zorder>
   <zorder>btnReady</zorder>
   <zorder>photo</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>photoPath</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
